test/index.js

```diff
@@ -24,7 +24,7 @@ describe('eslint-config-uphold', () => {
   it('should generate violations for incorrect code', async () => {
     const source = path.join(__dirname, 'fixtures', 'incorrect.js');
     const results = await linter.lintFiles([source]);
-    const rules = results[0].messages.map(violation => violation.ruleId);
+    const rules = results[0].messages.map((violation) => violation.ruleId);
```

Comment:
Do we want to add parentheses with a single arg?

Reply:
Related to https://github.com/uphold/eslint-config-uphold/pull/36#discussion_r565924451.

I guess we could raise this question to the team, personally I'm not opposed to require parenthesis, but since the default is not to use, I will change this accordingly.